src/ng-generate/components/form/generators/components/fields/structuredValue/files/__name@dasherize__.component.html.template

```diff
@@ -1,12 +1,15 @@
 <form [formGroup]="formGroup">
     <fieldset class="form-group">
-        <legend><%= options.fieldConfig.name %></legend>
+        <legend
+         data-test="<%= options.fieldConfig.name %>-date"
```

Comment:
Since it's not a form field, I think we can remove the test ID from here, or, if it's really necessary, we can use a different postfix, for example `label` or `group` instead of `date`.

Reply:
done